lib/BlueSCSI_platform_RP2040/BlueSCSI_platform_gpio.h

```diff
@@ -53,9 +53,9 @@
 #define SCSI_IN_REQ   9
 
 // Status LED pins
-//#define LED_PIN      25
-#define LED_ON()     (void)0 //cyw43_gpio_set(&cyw43_state, 0, true)  //sio_hw->gpio_set = 1 << LED_PIN
-#define LED_OFF()    (void)0 //cyw43_gpio_set(&cyw43_state, 0, false)  //sio_hw->gpio_clr = 1 << LED_PIN
+#define LED_PIN      25
+#define LED_ON()     platform_network_supported() ? cyw43_gpio_set(&cyw43_state, 0, true) : sio_hw->gpio_set = 1 << LED_PIN
+#define LED_OFF()    platform_network_supported() ? cyw43_gpio_set(&cyw43_state, 0, false) : sio_hw->gpio_clr = 1 << LED_PIN
```

Comment:
The lack of parentheses around each piece makes me nervous but that's all I have to complain about.

Reply:
I did consider that when reviewing this code before checking it in. My thinking at the time was:
1) It worked, I have tested this on both a Pico and Pico W
2) The code I was putting in a ternary was existing code that hadn't needed parentheses before
3) This isn't a #define with parameters (where it is always a good practice to put parentheses around the parameters as they could be arbitrarily complex)

I never remember the details of C operator precedence, so I went and had a look at the Wikipedia page. Interestingly it has a note that says that the middle of a ternary operator is already parsed as if parenthesized, so it would be redundant for the middle. If I am understanding the precedence correctly for the right hand side, even with the direct assignment being the same precedence as ternary, the associativity is right to left, so the assignment is handled before the ternary. So it can't be accidentally interpreted by the compiler as the following, which wouldn't work anyway:
(platform_network_supported() ? cyw43_gpio_set(&cyw43_state, 0, true) : sio_hw->gpio_set) = 1 << LED_PIN